.github/pull_request_template.md

```diff
@@ -1,67 +1,126 @@
+
+> E.g. Does the deployment setup need anything for this to work?
+
+## **3. The PR fixes the following GitHub issue / Jira task**
+
+<!-- Comment out the item which does not apply here.-->
+
+- [ ] GitHub issue (link):
+- [ ] Jira task (ID, `DDS-xxxx`):
+- [ ] The PR does not fix a specific GitHub issue or Jira task
+
+## **4. What _type of change(s)_ does the PR contain?**
+
+<!--
+- "Breaking": The change will cause existing functionality to not work as expected.
+- Workflow: E.g. a new github action or changes to this PR template. Anything that alters our or the codes workflow.
+-->
 
-- [ ] Bug fix
-  - [ ] Breaking: _Describe_
-  - [ ] Non-breaking
-- [ ] Documentation
 - [ ] New feature
-  - [ ] Breaking: _Describe_
+  - [ ] Breaking: _Please describe the reason for the break and how we can fix it._
+  - [ ] Non-breaking
+- [ ] Database change
+  - [ ] Migration _included in PR_
+  - [ ] Migration _not needed_
+- [ ] Bug fix
+  - [ ] Breaking: _Please describe the reason for the break and how we can fix it._
   - [ ] Non-breaking
 - [ ] Security Alert fix
+- [ ] Documentation
 - [ ] Tests **(only)**
 - [ ] Workflow
 
-_"Breaking": The change will cause existing functionality to not work as expected._
-
-# Checklist:
-
-## General
-
-- [ ] [Changelog](../CHANGELOG.md): New row added. Not needed when PR includes _only_ tests.
-- [ ] Database schema has changed
-  - [ ] A new migration is included in the PR
-  - [ ] The change does not require a migration
-- [ ] Code change
-  - [ ] Self-review of code done
-  - [ ] Comments added, particularly in hard-to-understand areas
-  - Documentation update
-    - [ ] Done
-    - [ ] Not needed
-
-## Repository / Releases
-
-- [ ] Blocking PRs have been merged
-- [ ] Rebase / update of branch done
-- [ ] PR to `master` branch (Product Owner / Scrum Master)
-  - [ ] The [version](../dds_web/version.py) is updated
-    - [ ] I am bumping the major version (e.g. 1.x.x to 2.x.x)
-      - [ ] I have made the corresponding changes to the CLI version
-  - Backward compatible
-    - [ ] Yes: The code works together with `dds_cli/master` branch
-    - [ ] No: The code **does not** entirely / at all work together with the `dds_cli/master` branch. _Please add detailed and clear information about the broken features_
-
-## Checks
-
-- [ ] CodeQL passes
-- [ ] Formatting: Black & Prettier checks pass
-- Tests
-  - [ ] I have added tests for the new code
-  - [ ] The tests pass
-- Trivy / Snyk:
-  - [ ] There are no new security alerts
-  - [ ] This PR fixes new security alerts
-  - [ ] Security alerts have been dismissed
-  - [ ] PR will be merged with new security alerts; This is why: _Please add a short description here_
+## **5. Checklist**
+
+<!-- Comment out the items which do not apply here.-->
+
+### **Always**
+
+<!-- Always go through the following items. -->
+
+- [Changelog](../CHANGELOG.md)
+  - [ ] Added
+  - [ ] Not needed (E.g. PR contains _only_ tests)
+- Rebase / Update / Merge _from_ base branch
```

Comment:
if _base_ branch means _dev_, I think we should have _dev_ here

Reply:
Base branch is the branch you forked from, so it could also be `master` for example. But we can add this info?